<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>579</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>579</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>579</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MenuPage">
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>579</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="WinPage">
     <widget class="QLabel" name="label_pic_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>579</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_game"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuSelect_Level">
    <property name="title">
     <string>Select Level</string>
    </property>
    <addaction name="actionLvl_1"/>
    <addaction name="actionLvl_2"/>
    <addaction name="actionLvl_3"/>
    <addaction name="actionLvl_4"/>
    <addaction name="actionLvl_5"/>
   </widget>
   <widget class="QMenu" name="menuLeaderBoard">
    <property name="title">
     <string>LeaderBoard</string>
    </property>
    <addaction name="actionSave_2"/>
    <addaction name="actionLoad_2"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuSelect_Level"/>
   <addaction name="menuLeaderBoard"/>
  </widget>
  <action name="actionNew_game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionEnd">
   <property name="text">
    <string>End</string>
   </property>
  </action>
  <action name="actionLvl_1">
   <property name="text">
    <string>Lvl 1</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionLvl_2">
   <property name="text">
    <string>Lvl 2</string>
   </property>
  </action>
  <action name="actionLvl_3">
   <property name="text">
    <string>Lvl 3</string>
   </property>
  </action>
  <action name="actionLvl_4">
   <property name="text">
    <string>Lvl 4</string>
   </property>
  </action>
  <action name="actionLvl_5">
   <property name="text">
    <string>Lvl 5</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
